<ui version="4.0" >
 <author>Dag Andersen &lt;danders@get2net.dk></author>
 <class>KPlato::WBSDefinitionPanelBase</class>
 <widget class="QWidget" name="KPlato::WBSDefinitionPanelBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WBSDefinitionPanelBase</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_2" >
     <property name="toolTip" >
      <string>Define code for the project node</string>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Project Code Definition</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QLabel" name="textLabel1_2" >
          <property name="text" >
           <string>Code:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="projectCode" />
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_3_3" >
          <property name="text" >
           <string>Separator:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="projectSeparator" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="toolTip" >
      <string>Define default code</string>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Default Definition</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Code:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="defaultCode" >
          <property name="sizeAdjustPolicy" >
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_3_2" >
          <property name="text" >
           <string>Separator:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="defaultSeparator" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="levelsGroup" >
     <property name="toolTip" >
      <string>Define separate codes for individual levels</string>
     </property>
     <property name="title" >
      <string>Use Levels Definition</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" colspan="3" >
       <widget class="QTableWidget" name="levelsTable" >
        <property name="columnCount" >
         <number>2</number>
        </property>
        <column>
         <property name="text" >
          <string>Code</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Separator</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QPushButton" name="removeBtn" >
        <property name="text" >
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="addBtn" >
        <property name="text" >
         <string>Add Level</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QSpinBox" name="level" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
